docs/metrics.md

```diff
@@ -425,15 +400,15 @@ Add time-based by invoking the following method:
 Function API
 
 ```python
-    def add_time(self, name: str, value: int or float, idx=None, unit: str = 'ms', dimensions: list = None,
+    def add_time(self, metric_name: str, value: int or float, idx=None, unit: str = 'ms', dimensions: list = None,
```

Comment:
The argument name defined in the [source code](https://github.com/pytorch/serve/blob/f1c22a55f3382a955525aab47317966f5c53f95a/ts/metrics/metric_cache_abstract.py#L134) is `name` and not `metric_name`.
The same applies to other functions of the API below.